api/server.go

```diff
@@ -0,0 +1,299 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"github.com/lbryio/lbry.go/errors"
+	"github.com/lbryio/lbry.go/util"
+	"github.com/lbryio/lbry.go/validator"
+	v "github.com/lbryio/ozzo-validation"
+
+	"github.com/fatih/color"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cast"
+)
+
+const authTokenParam = "auth_token"
+
+// Server HTTP Header Settings. Set on header if exists
+// ie. "Content-Type" - "application/json; charset=utf-8"
+// ie. "X-Content-Type-Options" - "nosniff"
+// ie. "X-Frame-Options" - "deny"
+// ie."Content-Security-Policy" - "default-src 'none'"
+// ie. "X-XSS-Protection" - "1; mode=block"
+// ie. "Server" - "lbry.io"
+// ie. "Referrer-Policy" - "same-origin"
+// ie. "Strict-Transport-Security" - "max-age=31536000; preload"
+// ie. "Access-Control-Allow-Origin" -"<header.Origin>"
+// ie. "Access-Control-Allow-Methods" - "GET, POST, OPTIONS"
+var HeaderSettings *map[string]string
```

Comment:
why is this a pointer? an uninitialized map already defaults to `nil`

Reply:
good catch!